scripts/rtx

```diff
@@ -103,11 +107,43 @@ function _itplugs() {
                 fi
                 ;;
             filemanager)
-                git clone https://github.com/nelu/rutorrent-filemanager filemanager >> /dev/null 2>&1
-                chmod -R +x /srv/rutorrent/plugins/filemanager/scripts
+                case $tag in
+                    3.*)
+                        svn co https://github.com/nelu/rutorrent-thirdparty-plugins/trunk/filemanager > /dev/null 2>&1
+                        chmod -R +x /srv/rutorrent/plugins/filemanager/scripts
+                        cat > /srv/rutorrent/plugins/filemanager/conf.php << FMCONF
+<?php
+\$fm['tempdir'] = '/tmp';
+\$fm['mkdperm'] = 755;
+// set with fullpath to binary or leave empty
+\$pathToExternals['rar'] = '$(which rar)';
```

Comment:
It's better to use `command -v` here instead of `which`. This is a posix command (shell built-in) works on every Linux distribution with no dependencies required. It also has less overhead than `which` and is not depreciated.